src/dotnet/HQ.Server/Services/PointServiceV1.cs

```diff
@@ -357,4 +360,21 @@ public async Task BackgroundAutoGenerateHolidayPlanningPointsV1(CancellationToke
         };
     }
 
+    public async Task BackgroundSendPointSubmissionReminderEmail(Period period, CancellationToken ct)
+    {
+        var startDate = DateOnly.FromDateTime(DateTime.UtcNow).GetPeriodStartDate(period);
+        var endDate = DateOnly.FromDateTime(DateTime.UtcNow).GetPeriodEndDate(period);
+        var points = _context.Points.Where(t => t.Date >= startDate && t.Date <= endDate);
+        var unsubmittedPoints = points.Where(t => !t.Completed);
+
+        var staffToNotify = await _context.Staff
+            .AsNoTracking()
+            .Where(t => t.EndDate == null && points.Where(x => x.StaffId == t.Id).Count() == 0 || unsubmittedPoints.Where(x => x.StaffId == t.Id).Count() > 0)
```

Comment:
Completed isn't used so we just need to check to see if a staff has 0 points in the database.